lib/parse/data/parseEmbedData.ts

```diff
@@ -54,7 +55,7 @@ const parseEmbedData = (
     // Override with values from config.
     ...(configTitle && { title: configTitle }),
     ...(configSubtitle && { subtitle: configSubtitle }),
-    ...(configAudioUrl && { url: configAudioUrl }),
+    ...(configAudioUrl && { url: generateAudioUrl(configAudioUrl) }),
```

Comment:
What's the difference between this "embed" data and the above parsed "audio" data?  (Wasn't expecting to see the mp3 url in 2 different places).